index.html

```diff
@@ -0,0 +1,898 @@
+                  63% OFF
+                </div>
+              </div>
+            </div>
+            <h4 class="story-title">
+              <a href="#">Dictanote Pro: Lifetime Subscription</a>
+            </h4>
+            <ul class="story-title">
+              <li class="color-gray"><s>$346</s>  $39</li>
+            </ul>
+          </div>
+          <div class="section-two-item">
+            <div class="section-five-item8 img-size-2 no-repeat ">
+              <div class="section-five-item-img">
+                <img src="./image/deals-ribbon.svg" alt="ribbon">
+                <div class="img-text">
+                  98% OFF
+                </div>
+              </div>
+            </div>
+            <h4 class="story-title">
+              <a href="#">The Official Unity Game Development Bundle</a>
+            </h4>
+            <ul class="story-title">
+              <li class="color-gray"><s>$796</s>  $29.99</li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </section>   
+  </main>  
+  <!-- Footer section -->
+    <footer class="footer-area">
+        <div class="footer-links ">
+          <ul class="list footer-list center ">
+            <li class="list-li" ><a  href="#" class="footer-social icon" ><i class="fab fa-facebook-square "></i></a></li>
+            <li class="list-li "><a href="#" class="footer-social icon"><i class="fab fa-twitter  "></i></a></li>
+            <li class="list-li"><a href="#" class="footer-social icon"><i class="fab fa-youtube "></i></a></li>
+            <li class="list-li"><a href="#" class="footer-social icon"><i class="fab fa-instagram "></i></a></li>
+            <li class="list-li"><a href="#" class="footer-social icon"><i class="fab fa-pinterest-p "></i></a></li>
+            <li class="list-li"><a href="#" class="footer-social icon"><i class="fas fa-wifi "></i></a></li>
+            <li class="list-li"><a href="#" class="footer-social icon"><i class="fas fa-envelope "></i></a></li>
+          </ul>
+          <ul class="list footer-list center">
+            <li class="list-li"><a href="#" class="footer-list-a" >EVENTS </a></li>
+            <li class="list-li"><a href="#" class="footer-list-a">ABOUT</a></li>
+            <li class="list-li"><a href="#" class="footer-list-a">ADVERTISE</a></li>
+            <li class="list-li"><a href="#" class="footer-list-a">JOBS</a></li>
+            <li class="list-li"><a href="#" class="footer-list-a">CONTACT</a></li>
+          </ul>
+        </div>
+        <div class="footer-legal" >
+            <div class=" legal-text center"><span ><img src="./image/tnw.svg" alt="logo" class="footer-legal-logo"></span><p class="footer-legal-text">© 2006–2020 The Next Web B.V.</p> <p class="footer-legal-text">Made with <span><i class="fas fa-heart"></i></span> in Amsterdam.</p><p><p class="footer-legal-text">Powered by</p> <span><img src="./image/stackpath.svg" alt="logo" class="footer-legal-logo2"></span></div>
+        </div>
```

Comment:
- [ ] these should be in one line on both large and medium screens, and 3 lines on small screens, yours changes on the medium screen